### PR TITLE
Helm chart

### PR DESCRIPTION
**Pull request by JeroenRijks**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -35,6 +35,13 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
     ]
 }
 
+// Vars for Kubernetes
+env.IDAM_API_BASE_URI = 'https://preprod-idamapi.reform.hmcts.net:3511'
+env.S2S_BASE_URI = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
```
Comment (linusnorton) [inline Jenkinsfile_CNP]: Can we use string interpolation:

``` 
env.S2S_BASE_URI = "http://rpe-service-auth-provider-${env}.service.core-compute-${env}.internal"
```

File: charts/em-npa/values.template.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    ENABLE_DB_MIGRATE: true
+    REFORM_SERVICE_NAME: em-npa
+    REFORM_TEAM: evidence
+    REFORM_SERVICE_TYPE: java
+    REFORM_ENVIRONMENT: preview
+    PACKAGES_NAME: em-npa
+    PACKAGES_PROJECT: evidence
+    PACKAGES_ENVIRONMENT: preview
+    ROOT_APPENDER: JSON_CONSOLE
```
Comment (linusnorton) [inline charts/em-npa/values.template.yaml]: That can go

File: Jenkinsfile_CNP [hunk 3]
```diff
@@ -35,6 +35,13 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
     ]
 }
 
+// Vars for Kubernetes
+env.IDAM_API_BASE_URI = 'https://preprod-idamapi.reform.hmcts.net:3511'
```
Comment (linusnorton) [inline Jenkinsfile_CNP]: Now that these vars are in here, can we remove them from output.tf?

File: Jenkinsfile_CNP (hunk 1 above)
Comment (JeroenRijks) [reply Jenkinsfile_CNP]: good shout, I'll give it a go after the show&tell :+1:

File: charts/em-npa/values.template.yaml (hunk 2 above)
Comment (JeroenRijks) [reply charts/em-npa/values.template.yaml]: :+1:

File: Jenkinsfile_CNP (hunk 3 above)
Comment (JeroenRijks) [reply Jenkinsfile_CNP]: depends, will we ever not need them in here, but need them in output.tf?
does output.tf export anywhere other than to Jenkins?

Comment (linusnorton) [reply Jenkinsfile_CNP]: I'm not sure. I read a confluence page a while back (wish I could find it again) that suggested it does output to other things but I've no idea what they would be. Maybe check with devops team

Comment (JeroenRijks) [reply Jenkinsfile_CNP]: I think it outputs to other apps if they refer to us in their infrastructure files, but that doesn't sound reasonable now that Im typing it

File: Jenkinsfile_CNP (hunk 1 above)
Comment (JeroenRijks) [reply Jenkinsfile_CNP]: Nooope, Jenkins didn't like it. Going back to how it was

**Pull request merged**